### ISSUE TITLE
[P2][Sourcing][Accessibility][Table]According to the ARIA specification, a grid must be labeled. The grid has no label. The heading in front of the grid is suitable as a label.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10283]
#### Is this a bug, enhancement, or feature request?
Bug
#### Briefly describe your proposal.
According to the ARIA specification, a grid must be labeled. The grid has no label. The heading in front of the grid is suitable as a label.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
tested on https://sap.github.io/fundamental-ngx/#/platform/table/basic#default

#### If this is a bug, please provide steps for reproducing it.
According to the ARIA specification, a grid must be labeled. The grid has no label. The heading in front of the grid is suitable as a label.

#### Is there anything else we should know?
Reported by customer

**Comment by nikolay-kolarov** [comment]
Hi,

There needs to be an API for the applicaitons to set the accessible name of the grid (which will result in aria-label or aria-labelldby attribute on the element with role grid).

Best Regards,
Nikolay

**Comment by droshev** [comment]
We need to downport this to `ng15`

**Comment by droshev** [comment]
fixed with `0.43.6`

**Comment by I543348** [comment]
ATL Re-verification needed

**Comment by droshev** [comment]
fixed with `0.43.7`

**Comment by sagar7259** [comment]
Hi Team,
Now the grid title/label is being associated on accessing the grid cells, hence this ticket can be closed.

Regards,
Sagar
